config-overrides.js

```diff
@@ -0,0 +1,29 @@
+const webpack = require("webpack");
+
+module.exports = function override(config, env) {
+    //do stuff with the webpack config...
+    config.resolve.fallback = {
+        ...config.resolve.fallback,
+        stream: require.resolve("stream-browserify"),
+        buffer: require.resolve("buffer"),
+    }
+    config.resolve.extensions = [...config.resolve.extensions, ".ts", ".js"]
+    config.plugins = [
+        ...config.plugins,
+        new webpack.ProvidePlugin({
+            process: "process/browser",
+            Buffer: ["buffer", "Buffer"],
+        }),
+    ]
+
+    // As of 2022/03/02 webpack config provided by react-scripts breaks cjs modules. Here's hack for now.
+    // Should be removed when https://github.com/facebook/create-react-app/pull/12021 will be merged
+    if (config.module.rules[1].oneOf[4].test.source === '\\.(js|mjs)$') {
+        config.module.rules[1].oneOf[4].test = /\.(js|mjs|cjs)$/;
+    } else {
+        throw new Error("Moera custom fix for cjs modules in react-scripts 5 is out of date." +
+            "Please remove or update it.");
+    }
+
+    return config
+}
```

Comment:
What about CORS header?

Reply:
Forgot about it. Will update the pr.

Reply:
Turns out, now it sets CORS headers by default https://github.com/facebook/create-react-app/blob/9673858a3715287c40aef9e800c431c7d45c05a2/packages/react-scripts/config/webpackDevServer.config.js#L47